doc/source/reference/command/return_codes/grn_operation_not_permitted.rst

```diff
@@ -0,0 +1,30 @@
+.. -*- rst -*-
+
+``GRN_OPERATION_NOT_PERMITTED``
```

Comment:
How about also adding the error code number like below?
I think users will look for the error code number because queries return it.

``-2: GRN_OPERATION_NOT_PERMITTED``